src/DDoSX/DomainClient.php

```diff
@@ -47,6 +49,22 @@ public function getByName($domainName)
         return $this->serializeDomain($body->data);
     }
 
+    /**
+     * @param Domain $domain
+     * @return SelfResponse
+     */
+    public function create(Domain $domain)
+    {
+        $response = $this->post('v1/domains', json_encode($this->friendlyToApi($domain, $this->requestMap)));
```

Comment:
Where is `requestMap` declared in this client class? It is declared in the admin class